cli/src/pcluster/templates/cdk_builder_utils.py

```diff
@@ -768,10 +768,13 @@ def _build_policy(self) -> List[iam.PolicyStatement]:
 
     def _generate_head_node_pass_role_resources(self):
         """Return a unique list of ARNs that the head node should be able to use when calling PassRole."""
+        resource_iam_path, _ = add_cluster_iam_resource_prefix(
```

Comment:
Minor: This function should be called as `get_cluster_iam_resource_prefix` instead of `add_cluster_iam_resource_prefix`